Documentation/cmdref/cilium-agent.md

```diff
@@ -57,6 +57,7 @@ cilium-agent [flags]
       --enable-endpoint-routes                        Use per endpoint routes instead of routing via cilium_host
       --enable-external-ips                           Enable k8s service externalIPs feature (requires enabling enable-node-port) (default true)
       --enable-health-checking                        Enable connectivity health checking (default true)
+      --enable-host-firewall                          Enable host network policies (default deny)
```

Comment:
I actually wanted to highlight that it will implement default-deny policies :laughing: 
I've removed it. We'll need to highlight that in the documentation anyway.